content/en/tracing/profiler/getting_started.md

```diff
@@ -202,6 +202,15 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
         profiler.WithEnv("<ENVIRONMENT>"),
         profiler.WithVersion("<APPLICATION_VERSION>"),
         profiler.WithTags("<KEY1>:<VALUE1>,<KEY2>:<VALUE2>"),
+        profiler.WithProfileTypes(
+          profiler.CPUProfile,
+          profiler.HeapProfile,
+          // The profiles below are disabled by default to keep overhead
+          // low, but can be enabled as needed.
```

Comment:
Thoughts on adding an empty line here for readability?

Reply:
Sure cbde153510b84b40721b3c825444162a75fe3de8 👍